src/System.Configuration.ConfigurationManager/src/System/Configuration/Internal/WriteFileContext.cs

```diff
@@ -4,6 +4,11 @@
 
 using System.IO;
```

Comment:
Could you upstream this change?

Reply:
@marek-safar the change has been upstreamed.